setup.py

```diff
@@ -31,11 +31,11 @@
             'html = aleph.ingest.html:HtmlIngestor',
             'djvu = aleph.ingest.djvu:DjVuIngestor',
             'img = aleph.ingest.image:ImageIngestor',
-            'email = aleph.ingest.email:EmailFileIngestor',
-            'mbox = aleph.ingest.email:MboxFileIngestor',
+            'email = aleph.ingest.email_:EmailFileIngestor',
```

Comment:
This is about requests looking for the builtin email module and finding the aleph email module. Apparently one doesn't name modules with the same name as builtin modules in python. Forget namespacing.

http://stackoverflow.com/questions/28685175/cannot-use-python-requests-module-if-file-is-named-email-py-or-logging-py

If I told you this was an adventure, I wouldn't be lying.